packages/utils/tokens/src/text-tokens.ts

```diff
@@ -1,15 +1,34 @@
-import { TextStyle } from 'react-native';
-import { IOperationSet } from '@uifabricshared/foundation-tokens';
+import { TextStyle, TextProps } from 'react-native';
 import { ITheme } from '@uifabricshared/theming-ramp';
+import { styleFunction } from '@uifabricshared/foundation-tokens';
 
 export interface ITextTokens {
   fontFamily?: TextStyle['fontFamily'] | string;
   fontSize?: TextStyle['fontSize'] | string;
   fontWeight?: TextStyle['fontWeight'] | string;
+  fontVariant?: string;

```

Comment:
Should this be instead something like fontVariant?: keyof ITypography['varaints']

Reply:
Also if we are adding the variant to the props then it might be good to create ITextVariantToken or something that can be included in the props interface and in the ITextTokens interface.

Reply:
I added keyof ITypography['variants'] to fontVariant and created ITextVariantTokens so it can be included in ITextTokens and in the props interface